torchvision/transforms/functional.py

```diff
@@ -82,6 +82,36 @@ def to_tensor(pic):
         return img
 
 
+def pil_to_tensor(pic, swap_to_channelsfirst=True):
+    """Convert a ``PIL Image`` to a tensor of the same type.
+
+    See ``AsTensor`` for more details.
+
+    Args:
+        pic (PIL Image): Image to be converted to tensor.
+        swap_to_channelsfirst (bool): Boolean indicator to convert to CHW format.
```

Comment:
after a second thought, let's remove this flag and always perform the transpose to CHW format.